INSTALL.md

```diff
@@ -56,9 +56,10 @@ Note, you may choose to install some or all of the ccl, numcosmo, and/or cluster
 Now, you can install CLMM and its dependencies as
 
 ```bash
-    pip install numpy scipy astropy matplotlib
+    pip install numpy scipy astropy matplotlib healpy
     pip install pytest sphinx sphinx_rtd_theme
     pip install jupyter  # need to have jupyter notebook tied to this environment, you can then see the environment in jupyter.nersc.gov
+    git clone https://github.com/LSSTDESC/qp.git; cd qp; python setup.py install; cd - # Install qp package
```

Comment:
Issuing `python setup.py install` is deprecated. Maybe we can change this part to `pip install git+https://github.com/LSSTDESC/qp.git` for those who don't want to clone the repo.